cloud: update doc for multi org (#10531)
pingcap/docs: tidb-cloud/manage-user-access.md

To access a cluster in a project under an organization, a user must be both a member of the organization and a member of the project. Organization owners can invite users to join the project to create and manage clusters in the project.

To view which organizations you belong to, perform these steps:

1. Click the account name in the upper-right corner of the TiDB Cloud console.
2. Hold the mouse over **Switch Organization**, and you can see all the organizations you belong to in the submenu.

To check which project you belong to, perform these steps:

1. Click the account name in the upper-right corner of the TiDB Cloud console.
2. Hold the mouse over **Switch Organization**, and click **Organization Settings**. The **Projects** tab is displayed by default.

## Switch organizations

If you are a member of multiple organizations, you can switch your account between organizations.

To switch organizations, perform these steps:

1. Click the account name in the upper-right corner of the TiDB Cloud console.

> 2. Hold the mouse over **Switch Organization**, and you can see all the organizations you belong to in the submenu.
> 3. Click the organization you want to switch to.

Author: ```suggestion
2. Hold the mouse over **Switch Organization**, and click the organization you want to switch to.
```

To access a cluster in a project under an organization, a user must be both a member of the organization and a member of the project. Organization owners can invite users to join the project to create and manage clusters in the project.

To view which organizations you belong to, perform these steps:

1. Click the account name in the upper-right corner of the TiDB Cloud console.

> 2. Hold the mouse over **Switch Organization**, and you can see all the organizations you belong to in the submenu.

Reviewer: Some people don't use mouses.
```suggestion
2. Move your cursor to **Switch Organization**, and you can see all the organizations you belong to in the submenu.
```

To check which project you belong to, perform these steps:

1. Click the account name in the upper-right corner of the TiDB Cloud console.
2. Click **Organization Settings**. The **Projects** tab is displayed by default.

> ## Switch organizations

Reviewer: ```suggestion
## Switch between organizations
```

If you are a member of multiple organizations, you can switch your account between organizations.

> To switch organizations, perform these steps:

Reviewer: ```suggestion
To switch between organizations, perform these steps:
```

1. Click the account name in the upper-right corner of the TiDB Cloud console.

> 2. Hold the mouse over **Switch Organization**, and click the organization you want to switch to.

Reviewer: ```suggestion
2. Move your cursor to **Switch Organization**, and click the organization you want to switch to.
```

7. After receiving this email, the user needs to click the link in the email to verify the identity, and a new page shows.

> 8. If the invited email address does not have a TiDB Cloud account, the user is directed to a sign up page to create an account. If the email address already has a TiDB Cloud account, the user is directed to the sign in page, and after sign-in, the account joins the organization automatically.

Reviewer: ```suggestion
8. If the invited email address has not been signed up for a TiDB Cloud account, the user is directed to the sign-up page to create an account. If the email address has been signed up for a TiDB Cloud account, the user is directed to the sign-in page, and after sign-in, the account joins the organization automatically.
```
